pkg/archive/archive_test.go

```diff
@@ -724,6 +725,77 @@ func TestTarUntar(t *testing.T) {
 	}
 }
 
+func TestTarWithOptionsChownOptsAlwaysOverridesIdPair(t *testing.T) {
+	origin, err := ioutil.TempDir("", "docker-test-tar-chown-opt")
+	require.NoError(t, err)
+
+	defer os.RemoveAll(origin)
+	filePath := filepath.Join(origin, "1")
+	// unprivilegeduser idPair
+	originalUID := 1000
```

Comment:
I've seen this user is explicitly added into the docker dev image, to me using root idPair wasn't explicit enough but maybe I'm wrong with that?